lib/WUI/link_content/prusa_link_api.cpp

```diff
@@ -117,6 +118,19 @@ namespace {
             return StatusPage(Status::NoContent, parser);
         }
     }
+
+    StatusPage print_file(char *filename, const RequestParser &parser) {
+        // if not exists
+        if (access(filename, R_OK) != 0) {
+            return StatusPage(Status::NotFound, StatusPage::CloseHandling::ErrorClose, parser.accepts_json);
+        }
+
+        if (wui_start_print(filename, true) == StartPrintResult::PrintStarted) {
+            return StatusPage(Status::NoContent, StatusPage::CloseHandling::ErrorClose, parser.accepts_json);
```

Comment:
Can I ask for a comment explaining the `ErrorClose` here? I know we agreed on that one OOB, but this _looks_ like a copy-paste error at first glance and someone might want to „fix“ it ;-)